internal/cli/filter_status.go

```diff
@@ -119,15 +119,15 @@ func (ff *filterFlags) orderOp() *pb.OperationOrder {
 func initFilterFlags(set *flag.Sets, ff *filterFlags, opts filterOption) {
 	f := set.NewSet("Filter Options")
 
-	if opts == fillterOptionAll || opts&filterOptionState != 0 {
+	if opts == filterOptionAll || opts == filterOptionState {
 		f.EnumVar(stateFlagVar(&ff.flagStatusFilter))
 	}
 
-	if opts == fillterOptionAll || opts&filterOptionPhyState != 0 {
+	if opts == filterOptionAll || opts == filterOptionPhyState {
 		f.EnumSingleVar(phyStateFlagVar(&ff.flagPhysState))
 	}
 
-	if opts == fillterOptionAll || opts&filterOptionOrder != 0 {
+	if opts == filterOptionAll || opts&filterOptionOrder != 0 {
```

Comment:
should this be `opts ==` too? Or did we mean for this to actually be a bitmask?

If its a bitmask we need to change our constant `iota` above to `1 << iota` I think.

Reply:
Oops, this needs to be the same change I made with the other if-statements and I had just missed it! I think the original intention was to use a bitmask but checking against `opts` like the change I made above works as intended too.